easy_phi/tests/run_tests.py

```diff
@@ -1,12 +1,46 @@
 # -*- coding: utf-8 -*-
-
+import tornado.httpserver
+import tornado.httpclient
+import tornado.ioloop
+import tornado.web
 import unittest
+import json
+from easy_phi import app as handlers
 
 class API_test(unittest.TestCase):
 
     def test_t(self):
         self.assertTrue(True)
 
+class TestTornadoWeb(unittest.TestCase):
+    http_server = None
+    response = None
+
+    def setUp(self):
+        application = tornado.web.Application([
+                (r'/api/v1/', handlers.PlatformInfoHandler),
```

Comment:
Not sure what exactly you're testing here. If it is URL handler itself, why do you have http server and IO loop? If it is URL schema and REST API, why do you deal with handler functions directly?